Makefile

```diff
@@ -494,12 +495,13 @@ ci-job-miri: ci-setup-miri
 
 ### ci-runner-github-qemu jobs:
 
+QEMU_COMMIT_HASH=a05f8ecd88f15273d033b6f044b850a8af84a5b8
 define ci_setup_qemu_riscv
 	$(call banner,CI-Setup: Build QEMU)
 	@# Use the latest QEMU as it has OpenTitan support
 	@printf "Building QEMU, this could take a few minutes\n\n"
-	@git submodule sync; git submodule update --init
-	@cd tools/qemu; ../qemu/configure --target-list=riscv32-softmmu --disable-linux-io-uring --disable-libdaxctl;
+  @git clone https://github.com/qemu/qemu ./tools/qemu 2>/dev/null || echo "qemu already cloned, checking out"
```

Comment:
Since we're minimizing, a slick thing to do might be to just pull down what's needed: https://stackoverflow.com/questions/31278902/how-to-shallow-clone-a-specific-commit-with-depth-1

Reply:
The indentation looks wrong here.

Reply:
Ah, it turns out this entire file uses tabs instead of spaces :/

Reply:
> Ah, it turns out this entire file uses tabs instead of spaces :/

`make` requires the indentation to be tabs.

Reply:
@ppannuto I tried the approach in the answer and did not get it to work. I may have just done something wrong. But the comments under the accepted answer suggest that GitHub repositories do not have the correct features enabled to support shallow cloning from a specific commit. Other comments suggest that this approach also creates a dependence on very new versions of git (e.g. sometime in 2019) which we might prefer to avoid anyways.

Feel free to add a commit that does this if you can get it to work though

Reply:
That sounds like it might be more fragile than it is worth for this use case